app/modules/import-export/module.php

```diff
@@ -208,8 +205,10 @@ private function render_import_export_tab_content() {
 
 			<?php
 			if ( $should_show_revert_section ) {
+				$admin_post_url = admin_url( 'admin-post.php?action=elementor_revert_kit' );
+				$nonced_admin_post_url = wp_nonce_url( $admin_post_url, 'elementor_revert_kit' );
 				$link_attributes = [
-					'href' => wp_nonce_url( admin_url( 'admin-post.php?action=elementor_revert_kit' ), 'elementor_revert_kit' ),
+					'href' => $this->maybe_add_referrer_param( $nonced_admin_post_url ),
```

Comment:
just extract all of it into a method that builds the URL.
When reading it, the developer doesn't care how the URL is gonna look like